README.md

```diff
@@ -0,0 +1,160 @@
+
+You can initialize the Replicate client using an API token or a configuration object.
+
+```kotlin
+val client = Replicate.client("your-api-token")
+
+// or
+
+val config = ReplicateConfig("your-api-token")
+val client = Replicate.client(config)
+```
+
+### Create a Prediction
+
+To create a prediction, you need to provide a `Predictable` instance.
+
+```kotlin
+suspend fun createPredictionExample() {
+    val predictable = PhotoBackgroundRemover(input = mapOf("image" to dataUri))
+    val predictionTask = client.createPrediction<String>(predictable)
+    println(predictionTask.result)
+}
+```
+
+### Get a Prediction
+
+Retrieve a prediction by its ID.
+
+```kotlin
+suspend fun getPredictionExample(predictionId: String) {
+    val predictionTask = client.getPrediction<String>(predictionId)
+    println(prediction.output)
+}
+```
+
+### Cancel a Prediction
+
+Cancel an ongoing prediction using its ID.
+
+```kotlin
+suspend fun cancelPredictionExample(predictionId: String) {
+    val result = client.cancelPrediction(predictionId)
+    if (result.isSuccess) {
+        println("Prediction canceled successfully")
+    } else {
+        println("Failed to cancel prediction: ${result.exceptionOrNull()}")
+    }
+}
+```
+
+### List Predictions
+
+To be implemented.
+
+```kotlin
+fun listPredictionsExample() {
+    val predictions = client.getPredictions()
+    println(predictions)
+}
+```
+
+
+### Await Prediction Result
+
+Awaits for the completion of a Prediction Task.
+
+```kotlin
+fun awaitPredictionTaskExample() {
+    val predictionTask = client.createPrediction<Any>(predictable)
+    val prediction = predictionTask.await()
+    println(prediction.output)
+}
+```
+
+## Example
+This example removes background from a photo. For the input image, you can use a remote url or a local file. However, 
+the file must be a bas64 encoded string as shown in the example.
+```kotlin
+suspend fun main() {
+    val imageFile = File(classLoader.getResource("image.jpg")!!.file)
+    val imageBytes = Files.readAllBytes(imageFile.toPath())
+    val encodedImage = Base64.encode(imageBytes)
+    val mimeType = "image/jpeg"
+    val dataUri = "data:$mimeType;base64,$encodedImage"
+
+    val client = Replicate.client("token")
+    val predictable = BgRemoval(input = mapOf("image" to dataUri))
+    val task = client.createPrediction<String>(predictable)
+    
+    val result = task.await()
+    println(result?.output)
+}
+```
+
+
+## Configuration
+
+You can configure the SDK behavior using the `ReplicateConfig` class.
+
+```kotlin
+data class ReplicateConfig(
+    val token: String,
+    val baseUrl: String = "https://api.replicate.com/v1"
+)
+```
+
+## API Reference
+Reference: https://replicate.com/docs/reference/http
+
+
+## Contributing
+
+Contributions are welcome! Please read our [contributing guidelines](CONTRIBUTING.md) for more details.
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.
```

Comment:
Do we have these files already?

Reply:
No, we don't. It is just a dummy. @Lamouresparus is working on it already.